subsys/bluetooth/host/conn_internal.h

```diff
@@ -9,6 +9,7 @@
  */
 typedef enum __packed {
```

Comment:
Why is this packed by the way?

Reply:
I think it is to make it an uint8_t instead of a uint32_t in the bt_conn definition, since it is declared as bt_conn_state_t state;
But not sure that is the original thought behind this.